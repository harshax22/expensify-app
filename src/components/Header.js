import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <header>
        <h1>Expensify</h1>
        <div>
    <NavLink to = "/" activeClassName = "is-active" exact = {true}>Visit the Dashboard Page!</NavLink>
    <NavLink to = "/create" activeClassName = "is-active">Create a new Expense!</NavLink>
    
     </div>
    </header>
   
   
);

export default Header;