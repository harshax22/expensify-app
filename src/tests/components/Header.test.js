import { shallow } from 'enzyme'
import React from 'react';
import Header from '../../components/Header';
import toJson from 'enzyme-to-json';

test('should render Header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
});