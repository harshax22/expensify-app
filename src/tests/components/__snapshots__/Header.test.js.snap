// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <div>
    <NavLink
      activeClassName="is-active"
      exact={true}
      to="/"
    >
      Visit the Dashboard Page!
    </NavLink>
    <NavLink
      activeClassName="is-active"
      to="/create"
    >
      Create a new Expense!
    </NavLink>
  </div>
</header>
`;
